public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -153,6 +169,7 @@ export function LokiContextUi(props: LokiContextUiProps) {
           />
         </Tooltip>{' '}
         Select labels to be included in the context query:
+        <LoadingPlaceholder text="" className={`${styles.loadingPlaceholder} ${loading ? '' : styles.hidden}`} />
```

Comment:
Wouldn't it be better to solve the height of div in `styles.textWrapper`  and then just do `{loading ? <LoadingPlaceholder className={styles.loadingPlaceholder}/> : null}`. Or set positioning in loadingPlaceholder to `absolute` with `top: 0` `right: spacing.1`.

Reply:
I just went with this approach because 
1. this is "easier" to implement
2. this is fast/more efficient in the browser. This is quite useless here, because there is only one component, but `{loading ? <LoadingPlaceholder className={styles.loadingPlaceholder}/> : null}` is basically mounting a component and unmounting it to the DOM, while my solution is just changing a CSS class.

Do you think my approach is okay, or should I do `{loading ? <LoadingPlaceholder className={styles.loadingPlaceholder}/> : null}`?

Reply:
👍